pkg/volume/azure_dd/mounter.go

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
What would you recommend as a different name for provisioner, mounter, and attacher?